ods_ci/tests/Resources/Files/llm/caikit_servingruntime.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       value: float32
     # TODO: This will eventually point to the official image
     # Note: This is currently the latest image, this can change frequently
-    image: quay.io/opendatahub/caikit-tgis-serving:pr-25
+    image: quay.io/opendatahub/caikit-tgis-serving:stable-4d0134e
```

Comment:
Is this still the correct image? @Jooho @Xaenalt

Reply:
I see a new one in quay - stable-1521ce7. Shall we use that one? @Jooho @Xaenalt